2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -295,9 +236,9 @@ ratios_matched <- ratios %>%
   
   # Add new columns with CL/NC matched information using the
   # matched_Heavy and matched_Light columns and the label swap information
-  # (rep1/2, CL=H; rep3/4 CL=L)
```

Comment:
This was wrong. Needed to be updated as per the update in the previous code chunk to account for your actual experimental design.